Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -9539,6 +9539,77 @@ a_Player:OpenWindow(Window);
 			},
 			Inherits = "cEntity",
 		},
+		cEnderCrystal =
+		{
+			Desc = [[
+				This class represents an ender crystal. This entity can be spawned by using {{cWorld#SpawnEnderCrystal_1|cWorld:SpawnEnderCrystal}}.
+			]],
+			Functions =
+			{
+				DisplaysBeam =
+				{
+					Returns =
+					{
+						{
+							Type = "boolean",
+						},
+						Notes = "Returns if the EnderCrystal shout display it's bream.",
+					},
+				},
+				GetBeamTarget =
+				{
+					Returns =
+					{
+						{
+							Type = "Vector3i",
+						},
+					},
+					Notes = "Returns the Block position where the beam is pointing",
+				},
+				SetBeamTarget =
+				{
+					Params =
+					{
+						{
+							Name = "BeamTarget",
+							Type = "Vector3i",
+						},
+					},
+				},
+				SetDisplayBeam =
+				{
+					Params =
+					{
+						{
+							Name = "ShowsBottom",
+							Type = "boolean",
+						},
+						Notes = "Sets if the EnderCrystal should broadcast it's bedrock base",
```

Comment:
it's -> its, add a full stop.
probably add a space in EnderCrystal, to Ender Crystal.
also perhaps "display" instead of "broadcast"?

Reply:
also this is SetDisplayBeam